pom.xml

```diff
@@ -526,4 +526,33 @@
 
     </plugins>
   </build>
+
+  <profiles>
+    <profile>
+      <id>nodeps</id>
+      <activation>
+        <activeByDefault>false</activeByDefault>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <version>2.19.1</version>
+            <configuration>
+              <runOrder>random</runOrder>
+              <reuseForks>false</reuseForks>
+              <excludes>
+                <exclude>Azure*Test</exclude>
+                <exclude>Amazon*Test</exclude>
+                <exclude>Kakadu*Test</exclude>
+                <exclude>Redis*Test</exclude>
+                <exclude>ControlPanelTest</exclude>
```

Comment:
Oh, so looking at the complete list, I think we can actually keep Ffmpeg, GraphicsMagick, ImageMagick, and OpenJpeg? Or maybe have another profile to include these...

Looks like we have 1) minimum of tests that can be executed in a blank box with Java + Maven, 2) a few more tests that require installation of open source / free tools & libraries (e.g. ffmpeg, graphicsmagick, imagemagick, openjpeg), 3) other tests that require licences or paid tools (e.g. kakadu, amazon, azure).

For the Travis CI tests, we can include 1) + 2). For 2) we'd need just to use a Ubuntu node, and [install some packages](https://docs.travis-ci.com/user/installing-dependencies/).

So do you think we should keep all tests executing by default, have a `nodeps` profile for developers and users to try (with your list above), and another profile for continuous integration (maybe `ci` ?) ?